Frameworks/CoreText/CTFont.mm

```diff
@@ -561,12 +655,32 @@ CGRect CTFontGetBoundingRectsForGlyphs(
 }
 
 /**
- @Status Stub
- @Notes Orientation parameter is not supported
+ @Status Caveat
+ @Notes  kCTFontVerticalOrientation returns an imprecise, equal advance for each glyph
 */
 double CTFontGetAdvancesForGlyphs(CTFontRef font, CTFontOrientation orientation, const CGGlyph glyphs[], CGSize* advances, CFIndex count) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    ComPtr<IDWriteFontFace1> fontFace1;
+    font->_dwriteFontFace.As(&fontFace1);
```

Comment:
same as above

---
In reply to: [80733082](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80733082) [](ancestors = 80733082)